selfdrive/car/toyota/carstate.py

```diff
@@ -29,7 +29,7 @@ def update(self, cp, cp_cam):
                         cp.vl["SEATS_DOORS"]['DOOR_OPEN_RL'], cp.vl["SEATS_DOORS"]['DOOR_OPEN_RR']])
     ret.seatbeltUnlatched = cp.vl["SEATS_DOORS"]['SEATBELT_DRIVER_UNLATCHED'] != 0
 
-    ret.brakePressed = cp.vl["BRAKE_MODULE"]['BRAKE_PRESSED'] != 0
+    ret.brakePressed = min(cp.vl["BRAKE_MODULE"]['BRAKE_PRESSED'] + cp.vl["BRAKE_MODULE2"]['BRAKE_PRESSED'], 1) != 0
```

Comment:
is the exiting signal not correct for the ES? if not, we should change the DBC so the signal name is correct

Reply:
Ah, sorry, forgot to add this to the PR description. It seems like BRAKE_MODULE2->BRAKE_PRESSED is a redundant signal, and ORing this with the original BRAKE_MODULE signal should make things safer. I've made a little change to improve readability.

Reply:
The existing signal is correct and the two match up 999,999 times out of 1,000,000. There was this one time about a year ago when OP did NOT disengage upon brake press, hence the fix.

Reply:
do you have a route where this happened?

Reply:
In any case, this change should be moved to a separate PR. The car port is good to merge once this is split out.

Reply:
Sounds good, I'll remove this change for now and open a separate PR for it